src/concepts/html/README.md

```diff
@@ -2,15 +2,24 @@
 description: The procedural macro for generating HTML and SVG
 ---
 
-# Using html!
+# Using `html!`
```

Comment:
Yes! I think we should adopt this as a convention accross all our documentation (using links to the relevant items in the API docs).